doc/functions.md

```diff
@@ -1,7 +1,7 @@
 # Functions
 
 Functions are the fundamental unit of program execution in any programming language.
-As in other languages, a Lean function has a name, can have parameters and take arguments, and has a body.
+As in other languages, a Lean function has a name, can have parameters and take arguments, have a body and return a value.
```

Comment:
Please revert this.

Reply:
It just seemed weird introducing functions without mentioning a return value, which is arguably the most important thing about a function.  How about this instead `have a body which returns a value` ?

Reply:
@lovettchris The body is the value.

Reply:
Great, can we say that then?

Reply:
@lovettchris What about learning more about Lean before trying to edit the existing documentation?

Reply:
I updated the text to say `As in other languages, a Lean function has a name, can have parameters and take arguments, declare a return type and have a body.` 

perhaps that satisfies both needs, that a "return type" is mentioned somewhere, but no mention of "return value" since body is the return value.